worknenjoy/gitpay
# Fix/handle error routes

## Description

> Successfully handles not a task route `/task/anything-here`

## Changes

- In `task.js`

## Issue

> Closes #502 

## Impacted Area

> None. Creates a new 404 component

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

- Type `/task/anything` in url and you should be redirected to 404 page

## Before
> Screenshot from the state before

## After
> Screenshot from the state after your Pull Request
![404-desktop](https://user-images.githubusercontent.com/52590791/78529540-71edd800-77ff-11ea-83fd-f1fb6ecf3842.png)
